libraries/AP_HAL_ChibiOS/hwdef/JHEM_JHEF405/JHEF-JHEF405PRO.config

```diff
@@ -0,0 +1,170 @@
+# Betaflight / STM32F405 (S405) 4.2.0 Nov 22 2020 / 18:38:23 (afdac08b3) MSP API: 1.43
+
+board_name JHEF405
```

Comment:
i don't think we want to include the betaflight config in our tree

Reply:
fixed